crates/shadowsocks-service/src/config.rs

```diff
@@ -872,7 +874,16 @@ impl Config {
 
         // Ext servers
         if let Some(servers) = config.servers {
-            for svr in servers {
+            // Filter out disabled servers
+            let enabled_servers = servers
+                .into_iter()
+                .filter(|ext_cfg| match ext_cfg.disable {
+                    Some(true) => false,
+                    _ => true,
+                })
+                .collect::<Vec<_>>();
+
+            for svr in enabled_servers {
```

Comment:
This `collect` is completely unnecessary. Just do

```rust
for svr in servers {
    if svr.disable.unwrap_or(false) {
        continue;
    }
}
```

Reply:
Sure